projects/eventonica/eventonica-part1-objects.md

```diff
@@ -39,52 +28,77 @@ Before you start coding, think about and write down the answers to these questio
 - What properties should each class have? Think about: What properties are needed to complete the assignment? What other properties would be helpful to include as well?
 - What methods should each class have?
 
-You may change your answers as you write the code, but it's good to have a general plan worked out before you code.
+You will likely change your answers as you write the code, but it's good to have a general plan worked out before you code.
 
 #### Starter Code
 
-Note:
-
 - Create a JavaScript file for your class(es); we recommend making a User and Event class above and outside of your EventRecommender class.
 - You will need to decide which arguments each function takes in.
 
 Start with the following code:
 
 ```javascript
-class EventRecommender {
-  constructor() {
-    // All main properties should go here.
+class Eventonica {
+  addEvent(/* arguments you decide go here */) {
+    // Adds a new Event
+  }
 
-    this.events = [];
-    this.users = [];
+  updateEvent() {
+    // Update existing Event
   }
 
-  addEvent() {
-    // Adds a new Event to the System
+  deleteEvent() {
+    // Deletes Event
+  }
+
+  findEventsByDate() {
+    // Return items in Event.all with a specified date
+  }
+
+  findEventsbyCategory() {
+    // Return items in Event.all with a specified category
   }
 
   addUser() {
-    // Adds a new User to the System
+    // Adds a new User
   }
 
-  saveUserEvent() {
-    // Allow users to save events to a personal Events array.
+  updateUser() {
+    // Update existing User
   }
 
   deleteUser() {
-    // Deletes a User from the system
+    // Deletes User
   }
+}
 
-  deleteEvent() {
-    // Deletes the Event from the system
+class Event {
+  static all = [];
```

Comment:
is `static` a JS thing?  I thought it was Java. I'm used to seeing `const` and `function`.

Reply:
Yep. Since this whole section is emphasizing OOP, it would actually probably be good to use it like other languages, since IIRC that was part of the point.

I'll add a link here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static

But maybe we should include that in one of the lessons? Not sure

Reply:
I don't think we need it in the lessons - it's pretty obscure when it comes to JS development in my opinion.